metrics/storage/webtooling.sh

```diff
@@ -47,10 +53,61 @@ help(){
 cat << EOF
 Usage: $0 <count>
    Description:
-        <count> : Number of containers to run.
+       <count> : Number of containers to run.
 EOF
 }
 
+clean() {
```

Comment:
maybe this function can be implemented here https://github.com/kata-containers/tests/blob/09a62469fe1343d2a34d0ff933b37b8176ab6dd3/lib/common.bash#L168, as it is kind of similar of what you are doing which I believe will help other tests that are using this function

Reply:
@GabyCT got it, removed from the script